src/server.c

```diff
@@ -4148,6 +4148,7 @@ void addReplyFlagsForKeyArgs(client *c, uint64_t flags) {
         {CMD_KEY_DELETE,          "delete"},
         {CMD_KEY_CHANNEL,         "channel"},
         {CMD_KEY_INCOMPLETE,      "incomplete"},
+        {CMD_KEY_VARIABLE_FLAGS,  "variable_flags"},
```

Comment:
same here.. this needs to be documented, and for that it needs to be useful.
the only way it'll be useful if if COMMAND GETKEYS will return flags..
let's drop it for now, and add it later.
maybe instead of deleting these lines, you should put them in a comment with some reasoning.